bundles/org.openhab.binding.knx/README.md

```diff
@@ -123,26 +121,57 @@ Note: After changing the DPT of already existing Channels, openHAB needs to be r
 *Attention:* Due to a bug in the original implementation, the states for DPT 1.009 are inverted (i.e. `1` is mapped to `OPEN` instead of `CLOSE`).
 A change would break all existing installations and is therefore not implemented.
 
-##### Channel Type "number"
+##### Channel Type `datetime`, `datetime-control`
+
+| Parameter | Description   | Default DPT |
+|-----------|---------------|-------------|
+| ga        | Group address | 19.001      |
+
+##### Channel Type `dimmer`, `dimmer-control`
+
+| Parameter        | Description                            | Default DPT |
+|------------------|----------------------------------------|-------------|
+| switch           | Group address for the binary switch    | 1.001       |
+| position         | Group address of the absolute position | 5.001       |
+| increaseDecrease | Group address for relative movement    | 3.007       |
+
+##### Channel Type `number`, `number-control` 
 
 | Parameter | Description   | Default DPT |
 |-----------|---------------|-------------|
 | ga        | Group address | 9.001       |
 
-Note: Using the Units Of Measurement feature of openHAB (Quantitytype) requires that the DPT value is set correctly.
+Note: The `number` channel has full support for UoM. 
+
+Using the Units Of Measurement feature of openHAB (QuantityType) requires that the DPT value is set correctly.
```

Comment:
If full name is needed, perhaps this should be introduced in the first of the two sentences?
```suggestion
Note: The `number` channel has full support for Units Of Measurement (UoM).

Using the UoM feature of openHAB (QuantityType) requires that the DPT value is set correctly.
```